target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts

```diff
@@ -343,15 +333,27 @@
 		pinctrl-0 = <&power_led_pin &sata_led_pin>;
 		pinctrl-names = "default";
 
-		power {
+		sata {
+                        label = "rango:white:sata";
+                        gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
+                };
+ 
+                wlan_2g {
+                        label = "rango:white:wlan_2g";
+                        gpios = <&gpio1 13 GPIO_ACTIVE_LOW>;
+                };
+ 
+                wlan_5g {
+                        label = "rango:white:wlan_5g";
+                        gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
+                };
+ 
+                power {
+ 			label = "rango:white:power";
```

Comment:
You still use spaces to indent instead of tabs, maybe you have to configure your editor to not replace tabs with spaces.

Also you might want to ask @kaloz if sorting by pins instead is even desired. Looking at a couple dts shipped with the kernel nothing is wrong as it currently is.

The wlan leds aren't fixed at this point, the pinctrl bits are needed, right?. Also the power led already works before, so just a code cleanup and not a fix. Me thinks you make things hard on yourself by making multiple logical changes in a single commit. I suggest to split this patch.